src/api/spec/lib/feature_switch/feature_spec.rb

```diff
@@ -1,25 +1,35 @@
 require 'rails_helper'
 
 RSpec.describe Feature do
-  let(:user) { create(:confirmed_user, :in_beta, login: 'Tom') }
-  subject { Feature.active?('bootstrap') }
+  subject { Feature.active?(:bootstrap) }
+
+  let(:is_beta) { false }
+
+  before do
+    allow(YAML).to receive(:load).and_return(yaml)
+    Feature.use_beta_features(is_beta)
```

Comment:
Why is better than stub? Why not create an user and pass it as argument?  Just in case!!!

https://martinfowler.com/articles/mocksArentStubs.html#TestIsolation